Dockerfile

```diff
@@ -1,9 +1,12 @@
 FROM docker:stable
+
+LABEL maintainer="team@lean-delivery.com"
+
 RUN apk add --no-cache git py-pip python-dev gcc musl-dev libffi-dev openssl-dev make
 RUN apk add --no-cache openssh-client
 
 RUN pip install --upgrade pip
-RUN pip install git+https://github.com/ansible/molecule.git@master
+RUN pip install molecule
 RUN pip install --upgrade ansible==2.5.* docker pyOpenSSL PyYAML pytest==3.7.4
```

Comment:
Do we still need to stick to pytest 3.7.4?

Reply:
No, Fixed